src/rlookup.c

```diff
@@ -141,6 +142,8 @@ RLookupKey *RLookup_GetKeyEx(RLookup *lookup, const char *name, size_t n, RLooku
     if (!key) {
       key = createNewKey(lookup, name, n);
     } else if ((key->flags & RLOOKUP_F_ISLOADED || key->flags & RLOOKUP_F_QUERYSRC) && !(flags & RLOOKUP_F_OVERRIDE)) {
+      // If the caller wanted to mark this key as explicit return, mark it as such even if we don't return it.
```

Comment:
Comment above
> // 1. if the key is already loaded, or it has created by earlier RP for writing, return NULL

Should adjust the comment to also mention when **no override?**

Something like
```
// 1. When no override, and if the key is already loaded, or it has been created by earlier RP for writing, return NULL
```